cluster_test.go

```diff
@@ -119,3 +126,133 @@ func TestListParams(t *testing.T) {
 	err = testClient.EventsV1.DeleteSink(*sink.Sid)
 	assert.Nil(t, err)
 }
+
+func createValidationServer(channel chan bool) *http.Server {
+	server := &http.Server{}
+	server.Handler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		url := r.Header["X-Forwarded-Proto"][0] + "://" + r.Header["X-Forwarded-Host"][0] + r.URL.RequestURI()
+		r.ParseForm()
+		params := make(map[string]string)
+		for k, v := range r.PostForm {
+			params[k] = v[0]
+		}
+		requestValidator := twilio.NewRequestValidator(os.Getenv("TWILIO_AUTH_TOKEN"))
+		channel <- requestValidator.Validate(url, params, r.Header["X-Twilio-Signature"][0])
+		defer server.Close()
+	})
+	return server
+}
+
+func createStudioFlowParams(url string, method string) *StudioV2.CreateFlowParams {
+	jsonStr := fmt.Sprintf(`{
+		"description": "Studio Flow",
+		"states": [
+		  {
+			"name": "Trigger",
+			"type": "trigger",
+			"transitions": [
+			  {
+				"next": "httpRequest",
+				"event": "incomingRequest"
+			  }
+			],
+			"properties": {
+			}
+		  },
+		  {
+			"name": "httpRequest",
+			"type": "make-http-request",
+			"transitions": [],
+			"properties": {
+			  "method": "%s",
+			  "content_type": "application/x-www-form-urlencoded;charset=utf-8",
+			  "url": "%s"
+			}
+		  }
+		],
+		"initial_state": "Trigger",
+		"flags": {
+		  "allow_concurrent_calls": true
+		}
+	  }`, method, url)
+
+	var definition interface{}
+	_ = json.Unmarshal([]byte(jsonStr), &definition)
+
+	params := &StudioV2.CreateFlowParams{
+		Definition: &definition,
+	}
+	params.SetFriendlyName("Go Cluster Test Flow")
+	params.SetStatus("published")
+	return params
+}
+
+func createStudioExecutionParams() *StudioV2.CreateExecutionParams {
+	executionParams := &StudioV2.CreateExecutionParams{}
+	executionParams.SetTo(to)
+	executionParams.SetFrom(from)
+	return executionParams
+}
+
+func executeFlow(t *testing.T, flowSid string) {
+	_, exeErr := testClient.StudioV2.CreateExecution(flowSid, createStudioExecutionParams())
+	if exeErr != nil {
+		t.Fatal("Error with Studio Execution Creation: ", exeErr)
+	}
+}
+
+func TestRequestValidation_GETMethod(t *testing.T) {
```

Comment:
Think all the implementation here can be extracted into a separate function where you just pass the `method` that you want to test.